components/switch/remote_transmitter.rst

```diff
@@ -119,6 +121,10 @@ Supported remote codes:
 
 Configuration variables:
 
+- **jvc**: Send a JVC IR code.
```

Comment:
Let's keep NEC at the top (most common protocol AFAIK). Maybe put it below panasonic?